test/gen_source_callbacks.rb

```diff
@@ -56,18 +56,16 @@ def parse_log(input_path, yaml_path = nil)
 
   File.open(input_path, 'r') do |file|
     file.each_line.map do |line|
-      # Line format support checks
-      match = line.match(REGEXT_PRETTY)
+      match = ( line.match(/(\S+): {(.*)/) or line.match(/(\S+): $/) )
```

Comment:
Why do we need that now?

Reply:
This enbled us to generate source for events with(out) payload data.

```bash
[22:10:37.154215000] (+?.?????????) event_2: { sf_3 = true }
[22:10:37.154215000] (+?.?????????) event_2:  
```

If the event have payload, it is further processed by this regex, so that can ba obtain the values 

```ruby 
REGEXT_PRETTY = /
 =\s            # We are interested to the right of the equal sign
 (
    ""|         # Empty string
    ".*?[^\\]"| # String who can contain space and quoted string
    [^\s,]+     # Anything except space and comma
 )
/x
```